README.md

```diff
@@ -0,0 +1,71 @@
+# vui-table
+
+This component contains [Sass mixins](http://sass-lang.com) and CSS that you
+can use to style tables for display of [tabular data](http://www.w3.org/TR/html5/tabular-data.html).
+
+For element layout concerns see [vui-grid-system](https://github.com/Brightspace/valence-ui-grid-system).
+For further information on this component and other VUI components, see the docs
+at [ui.valence.d2l.com](http://ui.valence.d2l.com/).
+
+## Usage
+To style tables, first include its `SCSS` file. Then, apply the mixin using the
+`vui-table` selector on the table element.
+
```

Comment:
As basic as it is, should probably include the install snippet (NPM and Bower) to be consistent with the other repos.